topics/proteomics/tutorials/ml-modeling-of-anti-cancer-peptides/tutorial.md

```diff
@@ -0,0 +1,223 @@
+
+
+
+> ### {% icon hands_on %} Hands-on: Calculating CTD descriptors for ACPs and non-ACPs
+>
+> 1. {% tool [PDAUG Sequence Property Based Descriptors](toolshed.g2.bx.psu.edu/repos/jay/pdaug_sequence_property_based_descriptors/pdaug_sequence_property_based_descriptors/0.1.0) %} with the following parameters:
+>    - {% icon param-file %} *"Input fasta file"*: `ACPs.fasta` (output of **Input dataset** {% icon tool %})
+>    - *"DesType"*: `CTD`
+>
+> 1. {% tool [PDAUG Sequence Property Based Descriptors](toolshed.g2.bx.psu.edu/repos/jay/pdaug_sequence_property_based_descriptors/pdaug_sequence_property_based_descriptors/0.1.0) %} with the following parameters:
+>    - {% icon param-file %} *"Input fasta file"*: `non_ACPs.fasta` (output of **Input dataset** {% icon tool %})
+>    - *"DesType"*: `CTD`
+>
+>
+{: .hands_on}
+
+
+## Preparing a traning data set
+
+We will combine the ACPs and non-ACPs data set as a single data frame and will add the class label. 
+
+### Adding class labels to the training data
+
+
+The class label usually describes samples from two different groups, in our case ACPs and non-ACPs.   In the case of binary classification, usually, samples are labeled as "0" or "1". 
+
+
+
+> ### {% icon hands_on %} Hands-on: Adding class labels to the data frames
+>
+> 1. {% tool [PDAUG Add Class Label](toolshed.g2.bx.psu.edu/repos/jay/pdaug_addclasslabel/pdaug_addclasslabel/0.1.0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file"*: `PDAUG Sequence Property Based Descriptors on data 1 - CTD (tabular)` (output of **Peptide Sequence Descriptors** {% icon tool %})
+>    - *"Class Label"*: `1`
+>
+> 2. {% tool [PDAUG Add Class Label](toolshed.g2.bx.psu.edu/repos/jay/pdaug_addclasslabel/pdaug_addclasslabel/0.1.0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file"*: `PDAUG Sequence Property Based Descriptors on data 2 - CTD (tabular)` (output of **Peptide Sequence Descriptors** {% icon tool %})
+>    - *"Class Label"*: `0`
+>
+>
+{: .hands_on}
+
+
+### Merging ACPs and non-ACPs samples to create a traning dataset
+
+In previous steps, we labeled positive data or ACPs as "1", and negative data or non-ACPs as "0", now we can combine these two data frames as one training dataset.  In this step, we will merge these data frames as a labeled training data set. 
+
+> ### {% icon hands_on %} Hands-on: Merging two data frames
+>
+> 1. {% tool [PDAUG Merge Dataframes](toolshed.g2.bx.psu.edu/repos/jay/pdaug_merge_dataframes/pdaug_merge_dataframes/0.1.0) %} with the following parameters:
+>    - {% icon param-files %} *"Input files"*: `PDAUG Add Class Label on data 3 - (tabular)` (output of **Add Class Label** {% icon tool %}), `PDAUG Add Class Label on data 4 - (tabular)` (output of **Add Class Label** {% icon tool %})
+>
+>
+>
+{: .hands_on}
+
+
+## Applying 6 different ML algorithms on the training data set
+
```

Comment:
abbreviation ML is not introduced, suddenly naming changes form machine learning to ML abbreviation

Reply:
Full name is added before the abbreviation